docker/ceph/centos/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN dnf config-manager --set-enabled powertools
 RUN pip3 install -U pip
 
 # Sanity checks:
-RUN pip3 install mypy tox
+RUN pip3 install jsondiff mypy tox
```

Comment:
Provided this dep. in `requirements.txt` (so tox will install it), do we really need it here?

Reply:
Right, I think its not needed here now. will remove it.